pkg/controller/product/mapping_rules.go

```diff
@@ -154,11 +166,17 @@ func (t *ThreescaleReconciler) createNewMappingRules(desiredList []capabilitiesv
 			return errors.New("product metric method ref for mapping rule not found")
 		}
 
+		lastAttribute := false
+		if spec.Last != nil {
+			lastAttribute = *spec.Last
+		}
+
 		params := threescaleapi.Params{
 			"pattern":     spec.Pattern,
 			"http_method": spec.HTTPMethod,
 			"metric_id":   strconv.FormatInt(metricID, 10),
 			"delta":       strconv.Itoa(spec.Increment),
+			"last":        strconv.FormatBool(lastAttribute),
```

Comment:
An alternative would be only specifying the 'last' parameter to 3scale api when it's been set to a non-nil value in the CR

Reply:
Updated createNewMappingRules method to only send "last" parameter to the 3scale API when it is set to a non-nil value